package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "hooks",
-  "version": "0.1.3",
+  "version": "0.2.1",
```

Comment:
Can you revert this back. This should be a part of the release and it should be in a separate PR :relaxed: